charts/incubator/wyoming-whisper/Chart.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: v2
+appVersion: "latest"
+dependencies:
+  - name: common
+    repository: https://library-charts.truecharts.org
+    version: 12.14.6
+deprecated: false
+description: Wyoming Whisper
+home: https://truecharts.org/charts/incubator/wyoming-whisper
+icon: https://truecharts.org/img/hotlink-ok/chart-icons/wyoming-whisper.png
+keywords:
+  - wyoming-whisper
+kubeVersion: ">=1.16.0-0"
+maintainers:
+  - email: info@truecharts.org
+    name: TrueCharts
+    url: https://truecharts.org
+name: wyoming-whisper
+sources:
+  - https://github.com/truecharts/charts/tree/master/charts/incubator/wyoming-whisper
+type: application
+version: 1.0.0
```

new charts always starts as this.
```suggestion
version: 0.0.1
```